rotkehlchen/tests/external_apis/test_coingecko.py

```diff
@@ -44,7 +44,7 @@ def test_asset_data(session_coingecko):
         identifier='yearn-finance',
         symbol='yfi',
         name='yearn.finance',
-        image_url='https://assets.coingecko.com/coins/images/11849/small/yfi-192x192.png?1598325330',  # noqa: E501
+        image_url='https://assets.coingecko.com/coins/images/11849/small/yearn-finance-yfi.png?1676174746',  # noqa: E501
```

Comment:
This is already fixed

Reply:
yes but it's in bugfixes and this is a develop PR. It's either this or merge bugfixes to develop, which I want to do but well ... takes time